src/lib/gundb/UserStorageClass.js

```diff
@@ -1660,67 +1662,82 @@ export class UserStorage {
    * @returns {Promise} Promise with an array of feed events
    */
   async getFeedPage(numResults: number, reset?: boolean = false): Promise<Array<FeedEvent>> {
+    let { feed, feedIndex, cursor, feedIds, wallet } = this
```

Comment:
destructuring `this` looks nice
but lets say you are now searching for all places where this.feed is  used. not possible is anymore. need to search for "feed" which is ambigous.

Reply:
ok

Reply:
it was just FYI. the issue is every where we destructure this.
you need to consider "good looking code", vs readable, searchable etc...